frontend/src/app/firewood/forests/firewood-rules/firewood-permit-rules.component.ts

```diff
@@ -26,12 +27,14 @@ export class FirewoodPermitRulesComponent implements OnInit {
     this.applicationRulesForm = new FormGroup({
       acceptRules: new FormControl()
     });
+    this.buttonDisabled = false;
   }
 
   ngOnInit() {
   }
 
   createApplication() {
+    this.buttonDisabled = true;
```

Comment:
We're gonna want to move this down to around line 47-48.. inside the `if (this.applicationRulesForm.valid) {` block. Otherwise there could be a scenario where the user clicks the button before having clicked the checkbox. They would then be unable to proceed to the checkout regardless of the state of the checkbox because the button is in a disabled state.

Reply:
moved